scripts/image_create_ubuntu-19.04.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash --login
+UBUNTU_URL="http://cloud-images.ubuntu.com/disco/current/"
```

Comment:
Can we use https rather than http for pulling things from the net.

Reply:
Sure, seems to work with https too. Adjusted for this and for Ubuntu 18.04